ZJU_19 [ADD] a new style of Dunhuang(敦煌风)

chkhu opened the pull request:
Dunhuang style fuses Chinese, Tibetan, Central Asian and Western elements. It features flowing silhouettes in brightly colored silk, accented by vibrantly patterned skirts/pants and decorative accessories. 
- image used to train the model
<img width="268" alt="image" src="https://github.com/modelscope/facechain/assets/109655311/89a5ced0-5f6c-41dd-a0d6-3ef6a60b1693">

- results
<img width="320" alt="image" src="https://github.com/modelscope/facechain/assets/109655311/75281aa1-89c3-4464-89b9-cd98100254a1">
<br>
<img width="320" alt="image" src="https://github.com/modelscope/facechain/assets/109655311/bb4784c3-9c0a-46d6-a184-cede1b8aaf83">
<br>
<img width="320" alt="image" src="https://github.com/modelscope/facechain/assets/109655311/f3171a56-20f6-4fb9-ba6a-59b0a2855bb2">
<br>
<img width="320" alt="image" src="https://github.com/modelscope/facechain/assets/109655311/5678c0d2-35ef-4dcf-b0b4-ce0ba8ca2518">
<br>
<img width="320" alt="image" src="https://github.com/modelscope/facechain/assets/109655311/9bbfeb53-3b99-47b3-aadf-2ae78986c3fa">
<br>
<img width="320" alt="image" src="https://github.com/modelscope/facechain/assets/109655311/85b54ee8-3d9a-4d11-ae18-015240767245">

EunTilofy commented:
I have made several tests for this style, and the result is as follows,
![0 4-1 0](https://github.com/modelscope/facechain/assets/57314416/239f2876-377a-4a50-9a75-7a420515162d)
![0 4-1 0(2)](https://github.com/modelscope/facechain/assets/57314416/53d889a9-10cf-4d25-b91f-f260ea49c4e5)
![0 45-0 95](https://github.com/modelscope/facechain/assets/57314416/2db72f40-94ba-44c8-a1ce-3e14a63f5879)
![0 45-1 1](https://github.com/modelscope/facechain/assets/57314416/71733001-7928-428e-b2bf-d5f8b1019642)
![0 45-1 05](https://github.com/modelscope/facechain/assets/57314416/d94b4b0a-c805-408e-83fa-e90cc900fde5)
![0 45-1](https://github.com/modelscope/facechain/assets/57314416/b742db25-5247-491e-88bd-a9c37438d7cb)
![生成结果0 5-1 05](https://github.com/modelscope/facechain/assets/57314416/369c0ad9-27fd-482f-9009-2472fb3b11fc)
![0 45-1 05-](https://github.com/modelscope/facechain/assets/57314416/d603c828-1e27-4ecd-82b0-7667974b1ed8)

chkhu commented:
In commit 8df099e, we:
- altered weight match to be 0.45:1.05, which is proven to be of the best quality and stability
- added some neg words like 'fused fingers' and 'watermark'
- replaced the preview image with a newer and better one that fits the size standard
